code/obj/machinery/computer/announcement.dm

```diff
@@ -244,6 +247,14 @@
 		req_access = list(access_ai_upload)
 		name = "AI Announcement Computer"
 
+	catering
+		req_access = list(access_bar) //chef gets bar access
```

Comment:
no they don't, bartender has only bar and chef has only kitchen, its just most kitchens are behind bar so the doors are both bar and kitchen access